plugins/link/SharedMemory.cpp

```diff
@@ -15,12 +15,21 @@
 #	include <errno.h>
 #endif
 
+#include <atomic>
+
 #include <cstring>
 
 #include <iostream>
 
+// One atomically accessed chunk of shared memory
+typedef uint32_t Chunk;
```

Comment:
```suggestion
using Chunk = std::uint42_t;
```

Reply:
Thanks for the C++ nits. I'm more used to C.